xml/System.Runtime.InteropServices/RuntimeInformation.xml

```diff
@@ -62,7 +62,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The property returns one of the following strings:  
+ The property returns a string that starts with the following and is usually followed by the version of the framework:  
```

Comment:
instead of "starts with the following and is usually followed by the version of the framework": "indicates the name of the .NET implementation and is usually followed by its version:"